packages/core/package.json

```diff
@@ -169,29 +163,30 @@
   },
   "scripts": {
     "----------------------------------------- static analysis -----------------------------------------": "",
-    "test-typings": "tsc --noEmit --emitDeclarationOnly false && tsc -b test/tsconfig.json",
+    "test-typings": "tsc --noEmit && tsc -b test/tsconfig.json",
     "----------------------------------------- tests ---------------------------------------------------": "",
-    "mocha": "mocha -r ./test/registerEsbuild",
-    "test-unit": "yarn mocha \"test/unit/**/*.test.[tj]s\"",
+    "mocha": "mocha -r ../../register-esbuild.js",
```

Comment:
Can we still have this in a `test` folder? Together with the ESM unit test?

Reply:
Moved